gcloud/datastore/connection.py

```diff
@@ -418,7 +416,7 @@ def save_entity(self, dataset_id, key_pb, properties,
            not passed in 'properties' no longer be set for the entity.
 
         :type dataset_id: string
-        :param dataset_id: The dataset in which to save the entity.
+        :param dataset_id: The id of the dataset in which to save the entity.
```

Comment:
Not that it matters but `id` and `ID` are being intermixed to describe the same thing.